newIDE/app/src/SceneEditor/index.js

```diff
@@ -386,7 +402,29 @@ export default class SceneEditor extends Component {
     done(true);
   };
 
-  _onRenameObject = (objectWithContext, newName, done) => {
+  //use this function to check if the new name can be used
```

Comment:
I think this comment is not necessary, _canObjectUseNewName is explicit enough with the arguments

Reply:
done :)